kwok/tests/usersignup_test.go

```diff
@@ -0,0 +1,47 @@
+package kwok
+
+import (
+	"fmt"
+	"log"
+	"testing"
+)
+
+var containerID string
+
+func TestExecContainerCommand(t *testing.T) {
+	var err error
+	imageID, err := PodmanBuildImage()
+	if err != nil {
+		t.Fatalf("Failed to build image: %v", err)
+	}
+	containerID, err = PodmanRunContainer(imageID)
+	if err != nil {
+		t.Fatalf("Failed to run container: %v", err)
+	}
+	defer func() {
+		if err := PodmanCleanup(containerID); err != nil {
+			t.Errorf("Failed to cleanup: %v", err)
+		}
+	}()
+
+	result, err := PodmanExecCommand(containerID, "/usr/local/bin/get-uid-map.sh")
+	if err != nil {
+		t.Fatalf("Failed to execute command: %v", err)
+	}
+	log.Println("Result of get-uid-map.sh: ", result)
+
+	if !validateMap(result) {
+		t.Errorf("Returned map was incorrect, got: %v", result)
+	}
+	log.Println("All tests Successful")
+}
+
+func validateMap(m map[string]int64) bool {
+	for i := 1; i <= 9; i++ {
+		user := fmt.Sprintf("user%d", i)
+		if val, ok := m[user]; !ok || int(val) < 52542471 || int(val) > 52542479 {
```

Comment:
What are those numbers? Can we make them constants of some sort to make the test more obvious?